.github/workflows/rustdoc.yml

```diff
@@ -30,10 +30,13 @@ jobs:
     - name: Build Documentation
       run: cargo doc --all --no-deps
 
+    - name: Add index redirection
+      run: echo "<meta http-equiv=\"refresh\" content=\"0; url=twt\">" > ./target/doc/index.html
+
```

Comment:
Adding this index.html under docs so we can link to the docs by `https://xithrius.github.io/twitch-tui/`

Reply:
Very cool.